docs/source/tools/sandbox.rst

```diff
@@ -99,6 +99,10 @@ with SHA-256) with the public key loaded from the given JWKS URL.
        secret = "not-safe-for-production"
    }]
 
+.. note:: In order to prevent man-in-the-middle attacks, in production it is highly recommended to use
+          TLS with server authentication as defined by `RFC2818 <https://tools.ietf.org/html/rfc2818>`__
+          for any request sent to the Ledger API.
```

Comment:
I don't think that pointing to the RFC is particularly valuable to the reader of this document. A pointer to the documentation of how to configure mTLS (or probably an example) is likely more valuable.